frontend/src/components/pages/admin/Admin.Users.tsx

```diff
@@ -37,54 +35,47 @@ export class AdminUsers extends Component<{}> {
-                        render: (t, r) => {
-                            if (r.internalIdentifier == api.userData?.user.internalIdentifier)
+                        size: 1,
+                        header: 'Identifier',
+                        accessorKey: 'internalIdentifier',
+                        cell: ({row}) => {
+                            if (row.original.internalIdentifier == api.userData?.user.internalIdentifier) {
                                 return (
                                     <span>
+                                        <>
                                         <Tooltip label="You are currently logged in as this user" placement="top" hasArrow>
-                                            <UserOutlined style={{ fontSize: '16px', padding: '2px', color: '#ff9e3a' }} />
+                                            <UserOutlined style={{fontSize: '16px', padding: '2px', color: '#ff9e3a'}}/>
                                         </Tooltip>{' '}
-                                        {t}
+                                        {row.original.internalIdentifier}
+                                        </>
                                     </span>
                                 );
-                            return t;
+                            }
+                            return row.original.internalIdentifier;
                         }
                     },
-                    { width: 1, title: 'OAuthUserID', dataIndex: 'oauthUserId', sorter: sortField('oauthUserId') },
-                    { width: 1, title: 'Roles', dataIndex: 'roles', render: (_text, user) => user.grantedRoles.map(r => r.role.name).join(', ') }, // can't sort
-                    { width: 1, title: 'Login', dataIndex: 'loginProvider', sorter: sortField('loginProvider') },
-                    { title: '', render: _r => <span></span> }
+                    { size: 1, header: 'OAuthUserID', accessorKey: 'oauthUserId' },
+                    { size: 1, header: 'Roles', accessorKey: 'roles', cell: ({row: {original: user}}) => user.grantedRoles.map(r => r.role.name).join(', ') }, // can't sort
+                    { size: Infinity, header: 'Login', accessorKey: 'loginProvider' },
                 ]}
-                // expandIconAsCell={false}
-                // expandIconColumnIndex={0}
-                expandRowByClick={true}
-                expandedRowRender={(user: UserDetails) => (
-                    <Accordion
-                        defaultIndex={0}
-                        items={
-                            user.grantedRoles.map(r => ({
-                                heading: r.role.name,
-                                description: <RoleComponent role={r.role} grantedBy={r.grantedBy}/>
-                            }))
-                        }
-                    />
-                )}
+                subComponent={({row: {original: user}}) =>   <Accordion
```

Comment:
Can we fix linting here? maybe run `npm run lint` manually if needed

Reply:
I think it's OK now.